tic-tac-toe-app/components/board.jsx

```diff
@@ -0,0 +1,72 @@
+const container = document.querySelector('.container')
+const game = new Game()
+
+const Board = () => {
+  const [box1, setBox1] = React.useState("")
+  const [box2, setBox2] = React.useState("")
+  const [box3, setBox3] = React.useState("")
+  const [box4, setBox4] = React.useState("")
+  const [box5, setBox5] = React.useState("")
+  const [box6, setBox6] = React.useState("")
+  const [box7, setBox7] = React.useState("")
+  const [box8, setBox8] = React.useState("")
+  const [box9, setBox9] = React.useState("")
+  
+  const move = (e) => {
+    
+    const boxStates = {
+      box1: setBox1,
+      box2: setBox2,
+      box3: setBox3,
+      box4: setBox4,
+      box5: setBox5,
+      box6: setBox6,
+      box7: setBox7,
+      box8: setBox8,
+      box9: setBox9,
+    }
+    const box = e.target.id
+    if(game.turn % 2 !== 0){
+      boxStates[box](<PlayerOne/>)
+      document.getElementById(box).removeAttribute("onClick");
+      console.log(box)
+      game.playerOne.makeMove(parseInt(box[3]) - 1)
+      console.log(game.board)
+      game.checkBoard(game.turn)
+      
+      game.turn += 1
+      return game.turn
```

Comment:
The if statements are nearly identical. It appears that the difference is only which box state is set. In that case, this can be refactored into something like:

```suggestion
      if(game.turn %2 !==0}{
      boxStates[box](<PlayerOne/>)
      } else {
      boxStates[box](<PlayerTwo/>)
      }
      
      document.getElementById(box).removeAttribute("onClick");
      //etc.
```